protokube/pkg/protokube/codecs.go

```diff
@@ -20,14 +20,15 @@ import (
 	"bytes"
 	"fmt"
 	"github.com/golang/glog"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/pkg/api/v1"
 	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/v1"
-	"k8s.io/kubernetes/pkg/runtime"
 
 	_ "k8s.io/kubernetes/pkg/api/install"
 )
 
 func encoder() runtime.Encoder {
+	// This encoder is kubernetes core, not kops encoder
 	yaml, ok := runtime.SerializerInfoForMediaType(api.Codecs.SupportedMediaTypes(), "application/yaml")
```

Comment:
@lavalamp this previously worked, and I am thinking that we may need to initialize both client-go and kops apis.

```
--- FAIL: TestBuildEtcdManifest (0.00s)
	integration_test.go:72: error marshalling to yaml: error encoding *v1.Pod: no kind is registered for the type v1.Pod
```

This is the error we are getting, and I am not certain if we need the kops api here as well.  @justinsb does protokube utilize types from the kops api?

@lavalamp if this code requires both the Pod type and kops api types - ideas on how to pull that on off?